client/js/utils.js

```diff
@@ -8,8 +8,10 @@
  *   Pablo Fernández <pablo.fernandez@ulpgc.es>
  * MIT Licensed
 
+  This module contains a set of util to be used from another modules.
```

Comment:
"This module contains a set of util to be used from another modules." ->
"This module contains a set of utils to be used from other modules."